build/nuke/Build.Steps.cs

```diff
@@ -172,20 +172,20 @@ partial class Build
                     .SetFramework(framework)
                     .SetOutput(TracerHomeDirectory / framework)));
 
-            // StartupHook is supported starting .Net Core 3.1.
-            // We need to emit StartupHook and ClrProfilerManagedLoader assemblies only for .NET Core 3.1 target framework.
+            // AutoInstrumentationStartupHook is supported starting .Net Core 3.1.
+            // We need to emit AutoInstrumentationStartupHook and AutoInstrumentationLoader assemblies only for .NET Core 3.1 target framework.
             DotNetPublish(s => s
-                .SetProject(Solution.GetProject(Projects.StartupHook))
+                .SetProject(Solution.GetProject(Projects.AutoInstrumentationStartupHook))
```

Comment:
Perhaps for a separate PR but internally I don't think that we need the prefix `AutoInstrumentation`

Reply:
Well, I have a problem with a good name for `OpenTelemetry.AutoIntrumentation.csproj`. To be clear, I have added prefixes everywhere. But I understand your concern. If you have an idea how to call it, I will be happy to update all of them.

Reply:
Let's keep as it is for now. Let's the rename settle a bit then revisit if we have a better solution.